test/sql/test_warehouse/R/test_warehouse

```diff
@@ -0,0 +1,22 @@
+-- name: test_show_warehouse_clusters
```

Comment:
better to keep the same order as the test cases.

Reply:
done